cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetector.java

```diff
@@ -194,9 +199,38 @@ private void checkWithDelay(Anomaly anomaly, long delay) {
       }
     }
 
+    /**
+     * Check whether the given anomaly is fixable. An anomaly is fixable if it (1) meets completeness requirements
+     * and (2) load monitor is not in an unexpected state.
+     *
+     * @param anomaly The anomaly to check whether fixable or not.
+     * @return true if fixable, false otherwise.
+     */
+    private boolean isFixable(Anomaly anomaly) {
+      String skipMsg = (anomaly instanceof GoalViolations) ? "goal violation fix" : "broker failure fix";
+      LoadMonitorTaskRunner.LoadMonitorTaskRunnerState unavailableState = ViolationUtils.isUnavailableState(_loadMonitor);
```

Comment:
This would be clearer to be:
```
LoadMonitorTaskRunner.LoadMonitorTaskRunnerState state =_loadMonitor.state();
if (AnomalyDetectorUtils.isLoadMonitorReady(state)) {
...
}
```